### PR TITLE
Graphicsmagick update

### PR DESCRIPTION
**Pull request by satmandu**
##
Builds:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`
##

##
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=graphicsmagick crew update \
&& yes | crew upgrade
```

### CODE REVIEW

File: packages/graphicsmagick.rb [hunk 1]
```diff
@@ -3,60 +3,122 @@
...
-  depends_on 'libwmf' # R
-  depends_on 'libx11' # R
+  depends_on 'libwebp' if %w[x86_64 aarch64 armv7l].include?(ARCH)
+  depends_on 'libwmf' if %w[x86_64 aarch64 armv7l].include?(ARCH)
+  depends_on 'libx11' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'libxau' # R
   depends_on 'libxcb' # R
   depends_on 'libxdmcp' # R
-  depends_on 'libxext' # R
+  depends_on 'libxext' if %w[x86_64 aarch64 armv7l].include?(ARCH)
```
Comment (uberhacker) [inline packages/graphicsmagick.rb:51]: Could all these `depends_on ...` lines be inside one conditional?
```
if %w[x86_64 aarch64 armv7l].include?(ARCH)
  depends_on ....
end
```

File: packages/graphicsmagick.rb [hunk 2]
```diff
@@ -67,7 +131,7 @@ def self.build
       --enable-static=no \
       --with-modules \
       --with-perl \
-      --with-x \
+      #{%w[x86_64 aarch64 armv7l].include?(ARCH) ? '--with-x' : ''} \
```
Comment (uberhacker) [inline packages/graphicsmagick.rb:134]: Is this line supposed to be commented?

Comment (uberhacker) [reply packages/graphicsmagick.rb:134]: Never mind.  I see the `#{...}`.

**Review by uberhacker — APPROVED** [review]

File: packages/graphicsmagick.rb (hunk 1 above)
Comment (satmandu) [reply packages/graphicsmagick.rb]: Done.

**Pull request merged**